topics/introduction/tutorials/galaxy-intro-short/tutorial.md

```diff
@@ -64,7 +64,7 @@ In Galaxy, *server* and *instance* are often used interchangeably. These terms b
 ## What does Galaxy look like?
 
 > <hands-on-title>Log in to Galaxy</hands-on-title>
-> 1. Open your favorite browser (Chrome, Safari or Firefox as your browser, not Internet Explorer!)
+> 1. Open your favorite browser (Chrome, Safari, Edge or Firefox as your browser, not Internet Explorer!)
```

Comment:
;) is there anyone that claims that Edge is their fav browser? ;)

Just kidding

Reply:
Ha! Maybe not favourite, but there are more using Edge than Firefox. https://gs.statcounter.com/browser-market-share

Reply:
What a world ...